Runtime/Shaders/ComputeShaderHelper.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using UnityEngine;
+
+namespace Unity.MeshSync {
+    internal class ComputeShaderHelper {
+
+        private const string SHADER_CONST_OUTPUT = "Output";
+
+        private readonly ComputeShader shader;
+        int kernelIndex;
+
+        readonly uint groupSizeX;
+        readonly uint groupSizeY;
+
+        Vector2Int maxTextureSize = new Vector2Int(1, 1);
+
+        public ComputeShaderHelper(ComputeShader shader, string kernelName) {
+            this.shader = shader;
+
+            kernelIndex = shader.FindKernel(kernelName);
+            shader.GetKernelThreadGroupSizes(kernelIndex, out groupSizeX, out groupSizeY, out var gsZ);
+        }
+
+        public RenderTexture RenderToTexture(Texture existingTexture) {
+            var renderTarget = existingTexture as RenderTexture;
+
+            // If there is an existing renderTexture, reuse it:
+            if (renderTarget == null ||
+                renderTarget.width != maxTextureSize.x ||
+                renderTarget.height != maxTextureSize.y) {
+                renderTarget = new RenderTexture(maxTextureSize.x, maxTextureSize.y, 32) {
```

Comment:
might want to release the existing one as well which may happen for example if `renderTarget.width != maxTextureSize.x`

Reply:
That will happen in the `SetTextureSafe` call.

Reply:
Hm, that can easily introduce memory leak bugs in the future when we don't call `SetTextureSafe` anymore for any reason.
Is it possible to put code to clear it here as well ?

Reply:
Done now